scipy/signal/tests/test_windows.py

```diff
@@ -213,5 +213,8 @@ def test_windowfunc_basics():
         window(6, *params, sym=False)
 
 
+def test_kaiser_derived():
+    signal.kaiser_derived(6)
```

Comment:
What is the point of this test?  The only reason to have a test that doesn't check that the return value against a known result is as a regression test for a call that used to, say, generate a seg. fault or incorrectly raise an exception.